lib/android.js

```diff
@@ -726,15 +726,14 @@ function findSDK(dir, config, androidPackageJson, callback) {
 		return callback(true);
 	}
 
-	const proguardPath = path.join(dir, 'tools', 'proguard', 'lib', 'proguard.jar'),
-		emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
+	const emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
 		result = {
 			path: dir,
 			executables: {
 				adb:       path.join(dir, 'platform-tools', 'adb' + exe),
-				emulator:  fs.existsSync(emulatorPath) ? emulatorPath : path.join(dir, 'tools', 'emulator' + exe)
+				emulator:  fs.existsSync(emulatorPath) ? emulatorPath : path.join(dir, 'emulator', 'emulator' + exe)
 			},
-			proguard: fs.existsSync(proguardPath) ? proguardPath : null,
+			proguard: null,
```

Comment:
Just a FYI, I'm pretty sure we never got around to using proguard. We had a ticket (https://jira-archive.titaniumsdk.com/TIMOB/TIMOB-14740) to auto-generate it the config file, but it turned out to be super complex as we'd need a list of any and all native classes being used and I think that includes modules. Oh well.